docs/docs/reference/cli.md

```diff
@@ -2677,7 +2677,7 @@ deletes temp database user
 **Examples**:
 
 ```bash
-v-add-database-temp-user wordress hestia_sso_user mysql
```

Comment:
Please also update v-delete-database-temp-user in /bin/ folder

Reply:
@jaapmarcus  done :)